geopandas/geodataframe.py

```diff
@@ -15,7 +18,7 @@
 DEFAULT_GEO_COLUMN_NAME = "geometry"
 
 
-def _ensure_geometry(data):
+def _ensure_geometry(data, crs=None):
```

Comment:
So if data is already of GeometryDtype, a passed crs gets ignored. This might be OK depending on what we decide for the behaviour, but let's at least make this explicit here (eg note in the docstring)?